src/psyclone/tests/psyGen_test.py

```diff
@@ -1551,7 +1551,7 @@ def test_haloexchange_node_str():
     invoke = psy.invokes.invoke_list[0]
     schedule = invoke.schedule
```

Comment:
Not the fault of this PR, but `pylint` reports
```
psyGen_test.py:67:0: W0611: Unused CWriter imported from psyclone.psyir.backend.c (unused-import)
psyGen_test.py:68:0: W0611: Unused FortranWriter imported from psyclone.psyir.backend.fortran (unused-import)
```
Would you mind improving this?